examples/next-openai-pages/package.json

```diff
@@ -9,8 +9,10 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@ai-sdk/react": "latest",
     "@ai-sdk/openai": "latest",
+    "@ai-sdk/react": "latest",
+    "@ai-sdk/ui-utils": "latest",
+    "@vercel/blob": "^0.23.4",
```

Comment:
Do we need to mention / update the README for deployments?

Reply:
done! updated the readme with an updated `deployment url` and `.env.local.example`